docs/content/docs/engines/hive.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Hive"
+weight: 2
+type: docs
+aliases:
+- /engines/hive.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Hive
+
+Table Store currently supports the following features related with Hive:
+* Create, drop and insert into table store tables through table store Hive catalog. Tables created in this way can also be read directly from Hive.
+* Register existing table store tables as external tables in Hive.
+
+## Version
+
+Table Store currently supports Hive 2.x.
+
+## Install
+
+{{< stable >}}
+Download [flink-table-store-hive-connector-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-table-store-hive-connector-/{{< version >}}/flink-table-store-hive-connector-{{< version >}}.jar).
+{{< /stable >}}
+{{< unstable >}}
+
+You are using an unreleased version of Table Store. The simplest way of building Table Store is by running:
+
+```bash
+mvn clean install -DskipTests
+```
+
+You can find the bundle jar in `"./flink-table-store-hive/flink-table-store-hive-connector/target/flink-table-store-hive-connector-{{< version >}}.jar"`.
+
+{{< /unstable >}}
+
+Copy table store Hive connector bundle jar to a path accessible by Hive, then use `add jar /path/to/hive-connector.jar` to enable table store support in Hive.
+
+## Using Table Store Hive Catalog
+
+By using table store Hive catalog, you can create, drop and insert into table store tables from Flink. These operations directly affect the corresponding Hive metastore. Tables created in this way can also be accessed directly from Hive.
+
+Execute the following Flink SQL script in Flink SQL client to define a table store Hive catalog and create a table store table.
+
+```sql
+-- Define table store Hive catalog
```

Comment:
Although it is explained above, I feel that it is still possible to confuse Flink CLI and Hive CLI.
I think it is better to re-emphasis on Flink SQL here. Like:
```
-- Flink SQL CLI
-- Define table store Hive catalog
```